File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner scanner=new Scanner(System.in);
        String str;
        int A=scanner.nextInt();
        int B=scanner.nextInt();
        //str=scanner.nextLine();
        int C=scanner.nextInt();
        int max=0;
        int array[]={A,B,C};

        int total;
        if((A==B)&&(B==C)&&(A==C)){
            System.out.println(10000+A*1000);
        }else if((A!=B)&&(B!=C)&&(A!=C)){
            for(int i=0; i<array.length;i++){
                if(array[i]>max){
                    max=array[i];
                }
            }
            System.out.println(max*100);
        }else{
            if((A==B)&&(B!=C)&&(C!=A)){
                System.out.println(1000+A*100);
            }else if((A!=B)&&(B==C)&&(C!=A)){
                System.out.println(1000+B*100);
            }else if((A!=B)&&(B!=C)&&(C==A)){
                System.out.println(1000+C*100);
            }
        }
        scanner.close();
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a,b,c = map(int,input().split())
aa=[a,b,c]
m=max(aa)

if a==b and b==c and c==a:
  print(10000+a*1000)
elif a==b and b!=c and c!=a: 
  print(1000+a*100)
elif b==c and c!=a and a!=b:
  print(1000+b*100)
elif c==a and b!=c and a!=b:
  print(1000+c*100)
else:
  print(m*100)